### PR TITLE
Update user guide for Tasklist

### PR DESCRIPTION
**Pull request by volodymyr-melnykc**
## Description

PR for https://github.com/camunda/camunda-docs/issues/3871.

It also includes general improvements for Tasklist user guide, where content was split in multiple pages for better readability and structuring.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/components/tasklist/userguide/using-tasklist.md [hunk 1]
```diff
@@ -1,113 +1,89 @@
...
-## Example use case
-
-When you've successfully logged in, you'll see a screen similar to the following:
+## Tasks overview
 
-![tasklist-start-screen](../img/tasklist-start-screen_light.png)
+Tasklist has two main pages:
 
-On the left side of the screen, you can see the list of tasks. On the right side of the screen, you can see details of the currently selected task.
+- Tasks page (_described here_).
```
Comment (christinaausley) [inline docs/components/tasklist/userguide/using-tasklist.md:27]: Are we linking to something here?

File: docs/components/tasklist/userguide/find-relevant-tasks.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+- All open tasks
+- Assigned to me
+- Unassigned
+- Completed tasks
+
+![tasklist-default-filters](img/task-filters/tasklist-default-filters.jpg "List of the default filters")
+
+## Create new filter
+
+To create a new filter, click on the option to apply a new filter. This action will open a filter dialog.
```
Comment (christinaausley) [inline docs/components/tasklist/userguide/find-relevant-tasks.md:30]: I've cleaned up some of the UI references, but there are a few cases where we say "click the X option" -- instead, I would just say "Click **Enter UI element**."

Comment (christinaausley) [reply docs/components/tasklist/userguide/find-relevant-tasks.md:30]: Similar to how you did in the example use case doc! 👍

File: docs/components/tasklist/userguide/find-relevant-tasks.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
+---
+id: find-relevant-tasks
+title: Find relevant tasks
```
Comment (christinaausley) [inline docs/components/tasklist/userguide/find-relevant-tasks.md:3]: Alternatively, couldn't we just name this page Filters or Using filters?

File: docs/components/tasklist/userguide/tasklist-example-use-case.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+---
+id: tasklist-example-use-case
```
Comment (christinaausley) [inline docs/components/tasklist/userguide/tasklist-example-use-case.md:2]: I almost feel like this is more of "Getting started with Tasklist" or even just "Getting started" in the sidebar -- I think an example use case is almost like a particular task tutorial that may belong in guides. WDYT?

File: docs/components/tasklist/userguide/using-tasklist.md [hunk 5]
```diff
@@ -1,113 +1,89 @@
...
+- Follow up date
 
-![tasklist-task-ordering](img/tasklist-task-ordering.png)
+<img src={TaskTileSpecification} className={styles.noShadow} alt="Task attributes" />
 
-### Assign tasks
+## Task details
 
-When no tasks are assigned to you, the list appears empty.
+Select a task from the list to view its details. It usually includes a form that has to be filled in and submitted to complete a task.
```
Comment (christinaausley) [inline docs/components/tasklist/userguide/using-tasklist.md:53]: Could link to forms documentation here.

File: docs/components/tasklist/userguide/using-tasklist.md [hunk 6]
```diff
@@ -1,113 +1,89 @@
 ---
 id: using-tasklist
-title: Overview and example use case
-description: "What you can do with Tasklist and an example use case."
+title: Overview
+description: "What you can do with Tasklist."
```
[the remark this answers is not in the document]
Comment (volodymyr-melnykc) [reply docs/components/tasklist/userguide/using-tasklist.md]: Agree! Updated.

File: docs/components/tasklist/userguide/using-tasklist.md (hunk 1 above)
Comment (volodymyr-melnykc) [reply docs/components/tasklist/userguide/using-tasklist.md]: No. I removed the note "_described here_"

File: docs/components/tasklist/userguide/find-relevant-tasks.md (hunk 3 above)
Comment (volodymyr-melnykc) [reply docs/components/tasklist/userguide/find-relevant-tasks.md]: Agree! Updated.

**Comment by volodymyr-melnykc** [comment]
@christinaausley I made updates (and removed some snapshots) based on your suggestions.

File: docs/components/tasklist/userguide/using-filters.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+![tasklist-filter-dialog-with-advanced-options](img/task-filters/tasklist-filter-dialog-with-advanced-options.jpg "Available filter attributes")
+
+After defining the rules, click **Apply** to apply the filter. The system will then filter the tasks according to your criteria, displaying only the relevant tasks.
+
+## Save filter for future use
+
+If you need to use the filter again, you can save it:
+
+1. Click **Save** filter.
```
Comment (christinaausley) [inline docs/components/tasklist/userguide/using-filters.md:48]: Last question 👍  Is the UI **Save** or **Save filter**?

**Comment by christinaausley** [comment]
Nice updates @volodymyr-melnykc! One UI question above, and this should be good 👍 

Do you only want these changes in the `next` version of docs, or should this be backported to the current and/or older versions as well?

File: docs/components/tasklist/userguide/using-filters.md (hunk 7 above)
Comment (volodymyr-melnykc) [reply docs/components/tasklist/userguide/using-filters.md]: @christinaausley The CTA is called **Save**. I pushed an update for this.

**Comment by volodymyr-melnykc** [comment]
> Do you only want these changes in the `next` version of docs, or should this be backported to the current and/or older versions as well?

@christinaausley These changes are only for the `next` version of docs, no need to backport.

**Review by christinaausley — APPROVED** [review]

**Pull request merged**